nixnet/_session/base.py

```diff
@@ -335,6 +335,37 @@ def disconnect_terminals(self, source, destination):
         """
         _funcs.nx_disconnect_terminals(self._handle, source, destination)
 
+    def write_state_lin_schedule_change(self, state_value):
```

Comment:
Like the other functions, I don't think we need "write_state" in the name.

So the question is what is a name that makes sense for just this function?

Would `change_lin_schedule` work?